### DISCUSSION TITLE
Table font size

### DISCUSSION THREAD

**Discussion by olivroy** [quarto-dev/quarto-cli#1150] [Feature Requests]
It might be a good idea to control table font size

In a chunk
```{r}
#| tbl-font-size: "80%"
knitr::kable(head(mtcars))
```

Sometimes for large table, it is common that the font size is smaller.

**Comment by mcanouil** [comment]
This would only work for markdown table though.
[`kableExtra`](https://github.com/haozhu233/kableExtra) can be used to customise your table in different format.
I believe it is better to use knitr/kableExtra, gt/gtExtras, ..., to handle table design rather than having a possibly confusing option that does work only for regular markdown, which might not be the most frequent in cell chunk.

**Reply by olivroy** [reply]
Yeah, that makes sense. Thanks

**Reply by mcanouil** [reply]
To note, there is an issue opened #1153 but not for the 1.0 release.

**Comment by mcanouil** [comment]
Closing in favour of #1153.
